Workbooks/AKS/Workload Details AKS Engine/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Workload Details",
+    "author": "Microsoft",
+    "galleries": [
```

Comment:
How did you test that this workbook works? Without manually passing the parameters that would be passed in an AKS Engine environment, you can only do this in production. But of course, we wouldn't test in production. That's why there is a 'isPreview' property. Set that to true and this workbook will get deployed under a feature flag.

Reply:
Done